.github/workflows/lint.yml

```
@@ -46,6 +46,203 @@ jobs:
+        run: charts/pawtograder/tests/alert-rules.sh
+
+  # Parse-check and lint the fenced bash blocks in docs/operations/*.md.
+  #
+  # Shell embedded in Markdown is executed by nothing, so it rots invisibly until
+  # an incident. This branch shipped an orphaned `while ... do` in a runbook block
+  # that nothing noticed until a `bash -n` sweep was added by hand. The sweep is
+  # cheap, so it is carried permanently. Read the script's header for what it
+  # deliberately does NOT verify -- it is a parse and lint gate, not a run of the
+  # procedures.
+  runbook-blocks:
+    runs-on: pawtograder-ci
```

actionlint (1.7.12)

[error] label "pawtograder-ci" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)